frontend/src/components/activePage/ActiveTemplates/ActiveTemplates.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC } from 'react';
+import { Collapse } from 'antd';
+import { CaretRightOutlined } from '@ant-design/icons';
+import TemplateHeader from '../TemplateHeader/TemplateHeader';
+import { ITemplate, ICourse } from '../utils/data';
+
+export interface IActiveTemplatesProps {
+  courses: Array<ICourse>;
+  templates: Array<ITemplate>;
+  destroyAll: () => void;
+  nested: boolean;
+}
+
+const ActiveTemplates: FC<IActiveTemplatesProps> = ({ ...props }) => {
+  const { templates, courses, destroyAll, nested } = props;
+  const { Panel } = Collapse;
+  return (
+    <Collapse
+      accordion
+      className="bg-gray-200 border-0 mx-10"
```

Comment:
If you look at the storybook, when in mobile view, the `mx-10` is too much. I would reduce it in that case